### ISSUE TITLE
Reaper64 download fails with No match found on URL

### ISSUE THREAD

**Issue by Daz-wallace** [autopkg/orbsmiv-recipes#9]
Error:
`        Error in local.munki.Reaper64-dataJAR: Processor: URLTextSearcher: Error: No match found on URL: https://www.reaper.fm/download.php`

It looks like this has changed from version 5 to version 6.

Pull request incoming, but might be worth a discussion over if v6 (as a major release) should be it's own recipe?

**Comment by apizz** [comment]
You beat me to it!  It appears in addition to the new version there is a slightly different regex between the 64bit 10.5-10.14 & 64bit 10.15

https://www.reaper.fm/files/6.x/reaper60_x86_64.dmg
vs.
https://www.reaper.fm/files/6.x/reaper60_x86_64_catalina.dmg

May want to include that somewhere ...

**Comment by Daz-wallace** [comment]
No worries, yea I did spot that and wasn't sure the best way to proceed. Ultimately they're your recipes to do fit but I guess maybe a new one for `Reaper64-catalina.download.recipe` ?

Both Apps are version 6.00.0 ( :/ ) but do have different checksums:
Non-Catalina version:
```
$ shasum -a 256 /Users/darren/Downloads/reaper60_x86_64.dmg 
589977a587aa510d7430308b09383551ab32506e7a1150e38ecff533ee4fc71b  /Users/darren/Downloads/reaper60_x86_64.dmg
$ shasum -a 256 /Volumes/REAPER_INSTALL_64/REAPER64.app/Contents/MacOS/REAPER 
3c0cfb9d34ca3ea85258fb4785547394a1f40a12fa50825adc0b79c2eaecc0b4  /Volumes/REAPER_INSTALL_64/REAPER64.app/Contents/MacOS/REAPER
```
Catalina version:
```
$ shasum -a 256 /Users/darren/Downloads/reaper60_x86_64_catalina.dmg 
49a17c631faa079cda2664e9fb0fd353cbd9e56c83f407202ddacd19e2363f32  /Users/darren/Downloads/reaper60_x86_64_catalina.dmg
$ shasum -a 256 /Volumes/REAPER_INSTALL_64/REAPER64.app/Contents/MacOS/REAPER 
77200393ed2690389b8ae872c82fd304ae14af9903eb3c696c0a8e302f6b81d8  /Volumes/REAPER_INSTALL_64/REAPER64.app/Contents/MacOS/REAPER
```

**Comment by Daz-wallace** [comment]
Or if the Catalina version is backwards compatible, then just move to that?

**Comment by Daz-wallace** [comment]
Oh, my dev colleague (Richard) ran both Apps through Taccy (https://eclecticlight.co/taccy-signet-precize-alifix-utiutility-alisma/) and the Catalina version has an additional entitlement and is Hardened and notarised, although these alone wouldn't be reasons you can't use the Catalina version on older OSes

<img width="592" alt="Catalina Version" src="https://user-images.githubusercontent.com/4480783/70309826-47b2ec00-1806-11ea-9080-1eb986bf625e.png">
<img width="592" alt="Non-Catalina version" src="https://user-images.githubusercontent.com/4480783/70309827-484b8280-1806-11ea-8176-545081657aab.png">

**Comment by orbsmiv** [comment]
Thanks for your work on this. I'd already made a couple of changes before I spotted these PRs (apologies!) so closed #10 on the basis that I went down the version-agnostic route.

Frustratingly, Cockos seem to be versioning their 6x macOS applications incorrectly, e.g. versions _6.01_ and _6.02_ (as advertised on their website) are being versioned as `6.10.0` and `6.20.0` respectively in their `CFBundleShortVersionString` and `CFBundleVersion` keys, which may make things messy later. I've dropped them a message about this but haven't received a reply as yet.
